qgreenland/config/datasets.yml

```diff
@@ -1034,3 +1034,20 @@ The images and data are produced in a consistent way that makes the Index time-s
         spatial time series, 2008-3007. Arctic Data
         Center. doi:10.18739/A29G5GD39.
       url: 'https://doi.org/10.18739/A29G5GD39'
+
+
+# TODO: More descriptive id? Are these borders, ...?
```

Comment:
We should resolve this TODO before merging; dataset IDs end up in the input cache and are rarely cleaned up. Speaking of which, we should probably eventually write a cleanup tool to remove any dataset IDs in the input cache that aren't in the config.

Reply:
That's a good idea. We're waiting for feedback from Karl at the moment.

Reply:
Created a task to make a script that will prune outdated datasets from the cache dir. This might be a good test-case for that, so I'm going to go ahead and merge this!